torch/testing/_internal/jit_metaprogramming_utils.py

```diff
@@ -229,8 +229,15 @@ def the_method({}):
     return {}
 '''
 
+def value_to_literal(value):
+    if isinstance(value, str):
+        # Quotes string and escapes special characters
+        return ascii(value)
+    else:
+        return str(value)
+
 def get_call(method_name, func_type, args, kwargs):
-    kwargs_str = ', '.join([k + '=' + str(v) for k, v in kwargs.items()])
+    kwargs_str = ', '.join([k + '=' + value_to_literal(v) for k, v in kwargs.items()])
```

Comment:
This didn't work with strings e.g. it would emit `norm=ortho` without any quotes.

Reply:
Nice fix.